src/modules/reserve-overview/ReserveConfiguration.tsx

```diff
@@ -53,16 +52,16 @@ export const ReserveConfiguration: React.FC<ReserveConfigurationProps> = ({ rese
           alignItems: 'center',
           gap: 6,
           flexWrap: 'wrap',
-          mb: reserve.isFrozen ? '0px' : '36px',
+          mb: reserve.isFrozen || reserve.symbol == 'AMPL' ? '0px' : '36px',
```

Comment:
I'd like to start seeing us veer away from these one-off asset symbol checks if we can help it.  I brought up a simple example of storing them as an object, but there are likely more elegant solutions.  It's just maybe not the most readable/maintainable to have these kind of statements scattered about the FE.

Reply:
This actually comes from #1288 which I based this branch on since they both update ReserveConfiguration

I totally agree, in general this is extremely ugly. For this particular case though, since it's the only asset across all markets with this warning there really isn't a much cleaner way to handle it. Wrapping it in a `warningAssets` variable would just add unncessary code since it would end up with same `symbol === 'AMPL'` check anyways.

Reply:
Would it make more sense to add in the check on the tooltip itself and return null `reserve.symbol !== 'AMPL` so it's scoped (at least for the warning)?  Just an idea..